Tasks/google-play-release/GooglePlay.ts

```diff
@@ -149,39 +87,34 @@ async function run() {
         // #7) Commit the edit transaction
 
         tl.debug(`Getting a package name from ${apkFile}`);
-        let packageName: string = manifest.package;
-        updateGlobalParams(globalParams, 'packageName', packageName);
+        const packageName: string = manifest.package;
+        googleutil.updateGlobalParams(globalParams, 'packageName', packageName);
 
         tl.debug('Initializing JWT.');
-        let jwtClient: any = new google.auth.JWT(key.client_email, null, key.private_key, GOOGLE_PLAY_SCOPES, null);
+        const jwtClient: any = googleutil.getJWT(key);
         globalParams.auth = jwtClient;
 
         tl.debug('Initializing Google Play publisher API.');
-        let edits: any = publisher.edits;
-        [edits, edits.apklistings, edits.apks, edits.tracks, edits.listings, edits.images, jwtClient].forEach(bb.promisifyAll);
+        const edits: any = googleutil.publisher.edits;
 
         tl.debug('Authorize JWT.');
-        await jwtClient.authorizeAsync();
+        await jwtClient.authorize();
 
         console.log(tl.loc('GetNewEditAfterAuth'));
         tl.debug('Creating a new edit transaction in Google Play.');
-        let currentEdit: Edit = await getNewEdit(edits, globalParams, packageName);
-        updateGlobalParams(globalParams, 'editId', currentEdit.id);
```

Comment:
This line is removed in the new code path, is this intentional?

Reply:
I see you folded it in to `getNewEdit`.

Reply:
Personally, I'd think we should leave `globalParams` alone, and have an explicit call to update this like what we used to have to avoid changing parameters silently.  That's a side effect.

Reply:
I agree.  I'll make the change.